src/replace_letters.py

```diff
@@ -0,0 +1,4 @@
+def replace_letters(word):
+    return word.translate(
+        str.maketrans("abcdefghijklmnopqrstuvwxyz", "zeeediiihooooonuuuuutaaaaa")
+    )
```

Comment:
This is a fine solution, but doesn't help you learn skills that readily apply to a board set of problems.

Better would be to address the comment I made 3 weeks ago.   Luckily, `maketrans` can also take a dictionary instead of 2 strings. I'd like to to programmatically create the needed dictionary with the comment I made being the rough start.